packages/terra-cell-grid/src/CellGrid.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './CellGrid.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The child Cells for be passed to the cell grid.
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Indicates the desired divider styles to apply to a cell grid and its children.
+   */
+  dividerStyle: PropTypes.oneOf(['none', 'vertical', 'horizontal', 'both']),
```

Comment:
Consider calling out the types in the comment above? Site does not show the values that this prop can take.

Reply:
Updated.